src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1309,4 +1309,17 @@ def process_remove_identity_namespace(cmd, namespace):
 def process_msi_namespace(cmd, namespace):
     get_default_location_from_resource_group(cmd, namespace)
     validate_tags(namespace)
+
+
+def process_gallery_image_version_namespace(cmd, namespace):
+    TargetRegion = cmd.get_models('TargetRegion')
+    if namespace.target_regions:
+        regions_info = []
+        for t in namespace.target_regions:
+            parts = t.split('=', 1)
+            if len(parts) == 1:
+                regions_info.append(TargetRegion(name=parts[0]))
+            else:
+                regions_info.append(TargetRegion(name=parts[0], regional_replica_count=int(parts[1])))
```

Comment:
There should probably be some kind of error handling here. If the right-hand side doesn't resolve to an int, the validator will give a cryptic error. it should probably give a usage error.